solr/Queries/SolrQuery.php

```diff
@@ -82,14 +82,14 @@ public function addFacetDateField($dateField) {}
      * @param string $value <p>
      * The value to use.
      * </p>
-     * @param string $field_override <p>
+     * @param string $field_override [Optional] <p>
```

Comment:
[Optional] can be removed here, actually. Optionality of a parameter is obvious from its default value. Stubs use `[optional]` in some other places because due to some reasons it's impossible to add a default value in signature, but it's not a case here